src/lib/snyk-to-html.ts

```diff
@@ -177,6 +186,62 @@ async function readInputFromStdin(): Promise<string> {
   });
 }
 
+interface ActionableRemediation {
+  upgradeFrom: string; // e.g. package@version
+  upgradeTo: string;   // e.g. package@version
+  vulns: Vuln[];
+  severity: string;
+  severityScore: number;
+}
+
+interface Vuln {
+  id: string;
+  title: string;
+  severity: string;
+}
+
+function getVuln(id, vulnerabilities: any): Vuln {
+  const vuln = vulnerabilities.find((v) => v.id === id);
+  return {
+    id: vuln.id,
+    title: vuln.title,
+    severity: vuln.severity,
+  };
+}
+
+function getSeverityScore(vulns: Vuln[]) {
+  return (vulns.reduce((acc, vuln) => acc + (severityMap[vuln.severity] + 1), 0));
```

Comment:
nice!
can we test it?

Reply:
Absolutely, doing that now!

Reply:
Tests added (and extant ones updated to add new CLI arg) 👍